corehq/apps/dashboard/views.py

```diff
@@ -129,6 +129,20 @@ def _get_default_tiles(request):
     def can_view_apps(request):
         return can_edit_apps(request) and has_privilege(request, privileges.PROJECT_ACCESS)
 
+    def can_view_users(request):
```

Comment:
Nitpick: this would be simpler if it started with
```
if not has_privilege(request, privileges.PROJECT_ACCESS):
    return false
```
and then the second return statement were just a set of ORs.